easybuild/easyblocks/l/libqglviewer.py

```diff
@@ -49,10 +52,29 @@ def configure_step(self):
     def sanity_check_step(self):
         """Custom sanity check for libQGLViewer."""
         shlib_ext = get_shared_lib_ext()
+        """
+        From version 2.8.0 onwards qt version also gets added to the lib file names.
+        """
+        if LooseVersion(self.version) < LooseVersion("2.8.0"):
+            custom_paths = {
+                'files': [('lib/libQGLViewer.prl', 'lib64/libQGLViewer.prl'),
+                          ('lib/libQGLViewer.%s' % shlib_ext, 'lib64/libQGLViewer.%s' % shlib_ext)],
+                'dirs': ['include/QGLViewer'],
+            }
+        else:
+            addition = ''
+            for dep in ['Qt5', 'Qt6']:
+                if get_software_root(dep):
+                    addition = dep.lower()
+                    addition = '-' + addition
+                    break
+            else:
+                raise EasyBuildError("Missing Qt5 or Qt6 dependency")
+            custom_paths = {
+                'files': [('lib/libQGLViewer'+addition+'.prl', 'lib64/libQGLViewer'+addition+'.prl'),
+                          ('lib/libQGLViewer'+addition+'.%s' % shlib_ext,
+                              'lib64/libQGLViewer'+addition+'.%s' % shlib_ext)],
+                'dirs': ['include/QGLViewer'],
+            }
```

Comment:
Sorry for nitpicking here, but i think deduplicating it a bit like this is nicer, and `suffix` seems a bit more appropriate name
```suggestion
        if LooseVersion(self.version) < LooseVersion("2.8.0"):
            suffix = ''
        else:
            for dep in ['Qt5', 'Qt6']:
                if get_software_root(dep):
                    suffix = '-' + dep.lower()
                    break
            else:
                raise EasyBuildError("Missing Qt5 or Qt6 dependency")
        custom_paths = {
            'files': [('lib/libQGLViewer'+addition+'.prl', 'lib64/libQGLViewer'+addition+'.prl'),
                      ('lib/libQGLViewer'+addition+'.%s' % shlib_ext,
                          'lib64/libQGLViewer'+addition+'.%s' % shlib_ext)],
            'dirs': ['include/QGLViewer'],
        }
```